charts/aws-efs-csi-driver/templates/controller-deployment.yaml

```diff
@@ -34,7 +34,7 @@ spec:
         {{- with .Values.nodeSelector }}
         {{- . | toYaml | nindent 8 }}
         {{- end }}
-      {{- if .Values.serviceAccount.controller.create }}
+      {{- if .Values.controller.create }}
```

Comment:
this is redundant now since the whole file is wrapped in if .Values.controller.create

Reply:
okay - I'll get rid of this